src/test/java/nl/tudelft/planningstool/core/AppTest.java

```diff
@@ -1,10 +1,12 @@
 package nl.tudelft.planningstool.core;
 
 import com.google.inject.Inject;
+import org.jukito.JukitoRunner;
 import org.junit.Test;
-import util.TestBase;
+import org.junit.runner.RunWith;
 
-public class AppTest extends TestBase {
+@RunWith(JukitoRunner.class)
+public class AppTest {
```

Comment:
`AppTest` should not inherit from `TestBase` as that loads the `DatabaseTestModule` causing the Persistence Unit to be loaded twice under the same name (should have thrown an exception, maybe it deadlocks because both units are loaded under the same thread which is a bizarre condition). Furthermore we don't need the `JPAInitializer` loaded eagerly, because the `PersistService` is already started by the `PersistFilter` on the context in `App`.